packages/deployer/src/build/telemetry.ts

```diff
@@ -84,25 +96,28 @@ export function getTelemetryBundler(
       }),
     ],
   });
+
+  return { bundle, externalDependencies };
 }
 
 export async function writeTelemetryConfig(
   entryFile: string,
   outputDir: string,
 ): Promise<{
   hasCustomConfig: boolean;
+  externalDependencies: Set<string>;
 }> {
   const result = {
     hasCustomConfig: false,
   } as const;
 
-  const bundle = await getTelemetryBundler(entryFile, result);
+  const { bundle, externalDependencies } = await getTelemetryBundler(entryFile, result);
 
   await bundle.write({
     dir: outputDir,
     format: 'es',
     entryFileNames: '[name].mjs',
   });
```

Comment:
This is better as treeshaking etc has kicked in 
```suggestion
  const {output} = await bundle.write({
    dir: outputDir,
    format: 'es',
    entryFileNames: '[name].mjs',
  });
  
  const externals = output[0].imports.filter(x => !x.startsWith('./'))
```

Reply:
I see now, thank you!